### ISSUE TITLE
Atom never detects that updates are available

### ISSUE THREAD

**Issue by tedivm** [atom/atom#18804]
### Description

My system never detects Atom updates and always requires me to manually check and update.

### Steps to Reproduce

1. Click "Check for Update"

**Expected behavior:**

It should say an update is available.

**Actual behavior:**

It says I have the current version (1.31.2 at the moment)

**Reproduces how often:**

100%

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

Atom    : 1.31.2
Electron: 2.0.7
Chrome  : 61.0.3163.100
Node    : 8.9.3

apm  2.1.1
npm  6.2.0
node 8.9.3 x64
atom 1.31.2
python 3.7.2
git 2.20.1

This is on OSX.

### Additional Information

While I did mention a specific version (1.31.2) this is a recurring issue for me.

**Comment by rsese** [comment]
Have you already tried this @tedivm?

https://discuss.atom.io/t/i-am-unable-to-update-to-the-latest-version-of-atom-on-macos-how-do-i-fix-this/40054

**Comment by tedivm** [comment]
I've confirmed that all of those folders are owners by the proper user (and already were).

That issue is a bit different than what I'm seeing- with that one a new update is detected but can't be downloaded. My issue is that the updates are never detected to begin with, even when clicking "check for update".

<img width="532" alt="screen shot 2019-02-06 at 11 24 29 am" src="https://user-images.githubusercontent.com/121709/52367829-cdf17c80-2a01-11e9-84b4-89d523258085.png">

**Comment by rsese** [comment]
Yeah I think I've seen those instruction help in slightly different cases than documented but thanks for trying :+1:

It sounds like you've had this issue for a while so I assume you've just been manually installing Atom when there's a new release and the last release you manually installed was 1.31.2?  Did updates  work at some point for you and then stopped working?

If you install a more recent version of Atom but not the most recent (so install say [1.33.1](https://github.com/atom/atom/releases/tag/v1.33.1)), can you update to 1.34.0 from there or does it also say "No update available"?

**Comment by tedivm** [comment]
Okay so my Atom install updated to 1.34, and then stalled again- it's been stuck there since.

**Comment by tedivm** [comment]
Weirdly enough it looks like the new version tried to partially install but then failed, as it's showing up as an option in the dropdown for opening files but the newer version isn't working-

<img width="570" alt="Screen Shot 2019-05-15 at 9 51 00 AM" src="https://user-images.githubusercontent.com/121709/57793790-05ee1700-76f7-11e9-8e35-f9aeb4cb563e.png">

**Comment by tedivm** [comment]
And as you can see here it thinks 1.34 was the most recent version-

<img width="532" alt="Screen Shot 2019-05-15 at 9 52 09 AM" src="https://user-images.githubusercontent.com/121709/57793833-2322e580-76f7-11e9-93f0-ee8cd35684ea.png">

**Comment by tedivm** [comment]
I'm also seeing this on two separate laptops (with slightly different version numbers), so my guess is that this is more widespread of an issue than you realize but that people just aren't noticing it.

**Comment by wbt** [comment]
I'm also seeing this on Windows, next to failures that show the current version as well as telling me the latest version is 1.38.2, and that updating might fix the problem - but updating only provides a link to the GitHub release page, and it's not at all clear that customizations will carry over to a new manual install.  
![image](https://user-images.githubusercontent.com/563406/60525911-4dd3f800-9cbd-11e9-873e-919503a91376.png)

**Comment by tofu-rocketry** [comment]
I'm getting this with 1.40.1 when 1.41 is actually available.

**Comment by wbt** [comment]
For historical perspective, issue #8265 seems to be the same basic issue, but has been locked down with instructions to open a new issue if reproducible.

**Comment by tedivm** [comment]
This never got fixed, but I switched to vscode last month.
